testsuite/unittests/test_gif_psc_exp.sli

```diff
@@ -0,0 +1,117 @@
+/*
+ *  test_gif_psc_exp.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+ /* BeginDocumentation
+Name: testsuite::test_gif_psc_exp - sli script for test of gif_psc_exp with
+      external DC current and spike generators
+
+Synopsis: (test_gif_psc_exp) run
+
+
+Description:
+
+ This testscript checks the correctness of gif_psc_exp neuron model. It
+ injects external DC current and also three spikes to neuron and measures
+ its spike times. The spike times should match the expected values. The
+ expected spike times are determined by the Python code used for the
+ publication: http://wiki.epfl.ch/giftoolbox
+
+Author:  March 2016, Setareh
+SeeAlso: gif_psc_exp, testsuite::test_gif_psc_exp_multisynapse
+*/
+
+(unittest) run
+/unittest using
+
+% neuron with default parameters should fire spikes at 43.5, 83.1 and 125.5 ms
+435  /t1_neuron1  Set 
+831  /t2_neuron1  Set
+1255 /t3_neuron1  Set
+
+% neuron with defined parameters should fire spikes at 20.9 and 79.7 ms
+209  /t1_neuron2  Set 
+797  /t2_neuron2  Set
```

Comment:
I think it would be very helpful if you would add an exact description of how you obtained these numbers (reference to the precise version of the GIF_Toolbox and the script used to generate these numbers). That will make it much easier to understand what is going on when some strange error crops up in five years time.